packages/nimble-components/src/menu-button/tests/menu-button.spec.ts

```diff
@@ -384,11 +367,12 @@ describe('MenuButton', () => {
             it('should open the menu and focus first menu item when the toggle button is clicked', async () => {
                 await connect();
                 const menuButton = value.getMenuButton(element);
+                const pageObject = new MenuButtonPageObject(menuButton);
```

Comment:
Seems like all the tests (in this diff, at least) are repeating these steps. Can we put them in a `beforeEach`?